tracer/src/Datadog.Trace/RemoteConfigurationManagement/RemoteConfigurationSettings.cs

```diff
@@ -26,8 +26,10 @@ public RemoteConfigurationSettings(IConfigurationSource configurationSource)
             AppVersion = configurationSource?.GetString(ConfigurationKeys.ServiceVersion);
 
             var pollInterval = configurationSource?.GetInt32(ConfigurationKeys.Rcm.PollInterval);
+
+            // need a way of know if we're in a test
             pollInterval =
-                pollInterval is null or <= 0 or > 5000
+                pollInterval is null // or <= 0 or > 5000
```

Comment:
I think you could just revert this change and set `DD_INTERNAL_RCM_POLL_INTERVAL` from your test?

Reply:
Thanks, I thought I'd revered this already. Fixed in ae8a05e44.